pkcs7/src/cryptographic_message_syntax2004.rs

```diff
@@ -0,0 +1,378 @@
+//! Selected structures from RFC5652
+
+use core::cmp::Ordering;
+use der::asn1::{BitString, OctetString, SetOf, SetOfVec, UIntBytes};
+use der::{Any, Choice, Decodable, Sequence, ValueOrd};
+use spki::{AlgorithmIdentifier, ObjectIdentifier};
+use x509::attr::AttributeTypeAndValue;
+use x509::ext::pkix::SubjectKeyIdentifier;
+use x509::name::Name;
+
+// Use 2004 suffix to distinguish from the enum in content_type.rs
+/// ContentInfo ::= SEQUENCE {
+///   contentType ContentType,
+///   content \[0\] EXPLICIT ANY DEFINED BY contentType }
```

Comment:
All ASN.1 code dumps need to be in code blocks. It doesn't look like this was done for any ASN.1 code in this crate.